src/realm/parser/driver.cpp

```diff
@@ -1586,15 +1586,38 @@ Mixed ParserDriver::get_arg_for_index(std::string i)
     switch (type) {
         case type_Int:
             return int64_t(m_args.long_for_argument(arg_no));
-            break;
         case type_String:
             return m_args.string_for_argument(arg_no);
-            break;
         default:
             throw InvalidQueryError("Invalid index type");
     }
 }
 
+double ParserDriver::get_arg_for_coordinate(std::string str)
```

Comment:
Why the `str` is passed by copy? I'd have `const std::string&`

Reply:
Done.